src/lib/UI/Module/ContentTree/NodeFactory.php

```diff
@@ -362,10 +377,23 @@ private function buildNode(
             }
         }
 
+        /** @var string $currentUserLanguageCode */
+        $currentUserLanguageCode = $this->userSettingService->getUserSetting('language');
```

Comment:
Doesn't `UserSettingService` return `UserSetting`? We should probably access its value here instead of casting to string.

Reply:
Actually, I forgot to delete this. 🤦‍♂️